### ISSUE TITLE
Does "--landscape-layers     - Comma separated list of landscape layers" work to limit the number of layouts?

### ISSUE THREAD

**Issue by jaimemiguel** [jjsullivan5196/wvkbd#92]
When I start wvkbd-mobintl with --landscape-layers full I still get many keyboards.
If I do not use landscape it seems to work correctly

Has anybody else seen this?
I am using the latest version.

Thank you.

**Comment by proycon** [comment]
Yes, it should work. I can't reproduce this issue, it works as expected. If it was an issue it may have been fixed in the meantime?
